WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -970,7 +963,7 @@ protected Address doInBackground(Double... args) {
         }
 
         protected void onPostExecute(@Nullable Address address) {
-            if (!isAdded() || address == null || address.getMaxAddressLineIndex() == 0) {
+            if (address == null || address.getMaxAddressLineIndex() == 0) {
```

Comment:
One more thing: This is pretty much the reason I separated this into its own PR and specifically pinged you. I want to make sure I get this right, so I never have to worry about it again.